addon/mixins/dirty-confirm-route.js

```diff
@@ -34,6 +34,9 @@ export default Ember.Mixin.create({
         case ROLLBACK: return true;
         case BUBBLE: return true;
       }
+    },
+    rollbackModel: function(model) {
```

Comment:
Do you need to pass in model? Is it ever going to be different than the above `var model = this.get('dirtyModel') || this.currentModel;`?